tests/test_PeakError.py

```diff
@@ -0,0 +1,64 @@
+import PeakError
+import pandas as pd
+import os
+
+
+def test_PeakErrorChrom():
+    exampleDataPath = '%s/inst/exampleData/' % os.getcwd()
+    peaksPath = '%sonechrompeaks.bed' % exampleDataPath
+    peaksdf = pd.read_csv(peaksPath, sep='\t', header=None, index_col=False)
+    peaksdf.columns = ['chrom', 'chromStart', 'chromEnd']
+
+    regionsPath = '%sonechromlabels.bed' % exampleDataPath
+    regionsdf = pd.read_csv(regionsPath, sep='\t', header=None, index_col=False)
+    regionsdf.columns = ['chrom', 'chromStart', 'chromEnd', 'annotation']
+
+    output = PeakError.errorChrom(peaksdf, regionsdf)
+
+    testfp = [1, 1, 1, 0]
+
+    assert testfp == output['fp'].tolist()
+
+    testtp = [0, 1, 1, 1]
+
+    assert testtp == output['tp'].tolist()
+
+
+def test_PeakError():
+    exampleDataPath = '%s/inst/exampleData/' % os.getcwd()
+    peaksPath = '%speaks.bed' % exampleDataPath
+    peaksdf = pd.read_csv(peaksPath, sep='\t', header=None, index_col=False)
+    peaksdf.columns = ['chrom', 'chromStart', 'chromEnd']
+
+    regionsPath = '%slabels.bed' % exampleDataPath
+    regionsdf = pd.read_csv(regionsPath, sep='\t', header=None, index_col=False)
+    regionsdf.columns = ['chrom', 'chromStart', 'chromEnd', 'annotation']
+
+    output = PeakError.error(peaksdf, regionsdf)
+
+    testfp = [1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 0, 1, 0]
```

Comment:
these test cases are copied from https://github.com/tdhock/PeakError/blob/master/tests/testthat/test-examples.R ?

Reply:
No, I noticed in https://github.com/tdhock/PeakError/blob/master/tests/script.R that you had issues using testthat so I didn't really look too much into those tests. I was already planning on writing some more tests, but getting it working was the main priority

Reply:
well anyway the tests pass in the R version so if you copied the C code it should work the same

Reply:
PeakError.c/h should be identical